superset/commands/database/create.py

```diff
@@ -118,25 +111,23 @@ def run(self) -> Model:
         ) as ex:
             event_logger.log_with_context(
                 action=f"db_creation_failed.{ex.__class__.__name__}",
-                engine=database.db_engine_spec.__name__,
+                engine=engine,
             )
             raise DatabaseCreateFailedError() from ex
 
-        if ssh_tunnel:
-            stats_logger.incr("db_creation_success.ssh_tunnel")
-
         return database
 
     def validate(self) -> None:
```

Comment:
Does it make sense to validate in here if 1) the DB has an SSH config and 2) the FF is disabled? This way we short-circuit early.

Reply:
Yeah, let me do it!